### ISSUE TITLE
Dashboard widget should show the actual mention message

### ISSUE THREAD

**Issue by jancborchardt** [nextcloud/spreed#4468]
Currently, the Dashboard widget only shows "You were mentioned" which is a bit low on info (especially because notifications show more of the message). The subline should show the actual message which mentions you so you can get some more info before clicking.

Conversation list | Widget
-|-
![talk notifications](https://user-images.githubusercontent.com/925062/97179864-9f00d880-1799-11eb-98e5-369b3b954440.png) |![Talk widget](https://user-images.githubusercontent.com/925062/97179862-9e684200-1799-11eb-98b9-762e44391f3b.png)

**Comment by nickvergessen** [comment]
This is not possible with the current api.
Also what should happen on multiple mentions?

**Comment by jancborchardt** [comment]
> This is not possible with the current api.

Yup, that’s what I understood, just opening here so we don’t forget about it. What would need to be changed where to make it work? In Talk I assume, not in Dashboard?

> Also what should happen on multiple mentions?

To keep it similar to what happens in the subline of the conversation list: The most recent mention would be shown shown.

**Comment by nickvergessen** [comment]
Then we can reuse the stored last mention id, and basically need to adjust the API to optionally load that comment instead of the last message id when getting the rooms. However since we at that point don't know if we will have a match for a mention or call, we wouldn't load the last message for the half empty content message. So I guess we would need to run 2 requests in that case, or we totally bloat up the API response and always return both.

**Comment by ChristophWurst** [comment]
As a user I would also prefer if *You were mentioned* would only be used for direct mentions and not `@all`.

**Comment by nickvergessen** [comment]
With #10250 this turned into a backend task.

The subline is created in https://github.com/nextcloud/spreed/blob/e116b345ab77d54c353a2957739f2c8b56f803b9/lib/Dashboard/TalkWidget.php#L178-L205

As mentioned above we simply can use the "last mentioned id" and start rendering that message instead of the "last message"
